hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -255,11 +257,12 @@ objects:
     namespace: splunk-forwarder-operator
   spec:
     clusterDeploymentSelector:
-      matchLabels:
-        api.openshift.com/managed: "true"
-        api.openshift.com/environment: production
       matchExpressions:
+        - { key: api.openshift.com/managed, operator: In, values: ["true"] }
+        - { key: api.openshift.com/environment, operator: In, values: ["production"] }
         - { key: api.openshift.com/noalerts, operator: NotIn, values: ["true"] }
+        - { key: ext-managed.openshift.io/noalerts, operator: NotIn, values: ["true"] }
```

Comment:
What's this label and how's it different from the noalerts above it?

Reply:
Before we had a CRD for PD we only had noalerts for excluding a cluster from sending things to PD or DMS.  We asked SDA to create a way for cicd team to add a label to CD's so we could remove our hack, a cronjob on master nodes.  A general solution was put in place that "external" labels could be set via api.  So we support both these labels now in PD and DMS, should for splunk too.

Reply:
ack, thx for the context